GitUI/CommandsDialogs/RevisionFileTreeControl.cs

```diff
@@ -403,15 +431,32 @@ private void tvGitTree_ItemDrag(object sender, ItemDragEventArgs e)
         {
             if (e.Item is TreeNode { Tag: GitItem gitItem })
             {
-                StringCollection fileList = new();
-                var fileName = _fullPathResolver.Resolve(gitItem.FileName);
+                string tempFilePath;
+                try
+                {
+                    tempFilePath = SaveItemToTempFile(gitItem);
+                }
+                catch (UnauthorizedAccessException)
+                {
+                    MessageBox.Show(_cantSaveToTemp.Text, TranslatedStrings.Error, MessageBoxButtons.OK, MessageBoxIcon.Error);
+                    return;
+                }
+                catch (FileNotFoundException)
+                {
+                    MessageBox.Show(_cantExtractTempArchive.Text, TranslatedStrings.Error, MessageBoxButtons.OK, MessageBoxIcon.Error);
+                    return;
+                }
```

Comment:
There are more types of possible exceptions. And this mapping to messages might match often but not always.
Usually, the exception text is more informative than these slightly generic messages.
You could even let the exception simply bubble up to the `BugReportInvoker`, i.e. without catching.
In order to suppress bug reports, the `Exception` should be wrapped into a `UserExternalOperationException`.

Reply:
Any issues would very likely be user-level failures (i.e. some the user will be in a position to resolve), and must be reported via the BugReporter

Reply:
Thanks both, I can see your point, am not catching these locally now